practiceCourseGenerator/practiceCourseGenerator.py

```diff
@@ -246,10 +278,15 @@ def setupLogger(logdate):
 writeCsvFile('enrollments', outputDirectory, userDictionaries)
```

Comment:
Also, rather than using the strings "enrollments", "courses", and "users", look into using an enumeration instead.